sceptre/sageit/templates/accounts.yaml

```diff
@@ -56,7 +58,8 @@ Resources:
           -
             Effect: Allow
             Principal:
-              Service: ec2.amazonaws.com
+              - Service: ec2.amazonaws.com
+              - AWS: !GetAtt AWSIAMGithubActionUser.Arn
```

Comment:
It looks wrong to add a user to a policy meant for an EC2 instance.  I think you just create a new user in a separate template.  I think PR https://github.com/Sage-Bionetworks/aws-infra/pull/275/files will help you easily create a service user.

Reply:
OK, removed user from account.yaml and moved to config using the template you mentioned.